scripts/Packages/Package_Gullwing__QFP_SOIC_SO/Readme.md

```diff
@@ -23,47 +23,60 @@ FileHeader:
 - pin count, 0 means no pins in that direction (`num_pins_x`, `num_pins_y`) {int}
-- Exposed pad:
-  - Size of package exposed pad or slug (`EP_size_x`, `EP_size_y`) {dimension}
-  - Optional the size of the footprint pad (`EP_size_x_overwrite`, `EP_size_y_overwrite`) {float}
-  - Optional the size of the mask cutout (`EP_mask_x`, `EP_mask_y`) {float}
-  - Paste can be split into regular grid with (`EP_num_paste_pads`) {[int (x), int (y)]}
-    - Paste coverage multiplicator (`EP_paste_coverage`) {float (0..1)}
-  - Thermal vias (`thermal_vias`) with following sup parameters
-    - Number of vias generated in the regular grid (`count`) {[int (x), int (y)]}
-    - Final hole size (`drill`) {float}
-    - Optional grid (`grid`) {[float (x), float (y)]}
-    - Optional paste coverage overwrite (`EP_paste_coverage`) {float (0..1)}
-    - Should paste avoid vias? (`paste_avoid_via`) {bool}
-      - Can lead to math exceptions (reduce paste coverage)
-      - Clearance between via hole and paste (`paste_via_clearance`) {float}
-    - Number paste pads
-      - given as count value (`EP_num_paste_pads`) {[int (x), int (y)]}
-      - More specific as between vias and outside ring (`paste_between_vias`, `paste_rings_outside`)
+
+
+### Exposed pad handling:
+- Size of package exposed pad or slug (`EP_size_x`, `EP_size_y`) {dimension}
+- Optional the size of the footprint pad, equal to nominal EP_size (`EP_size_x_overwrite`, `EP_size_y_overwrite`) {float}
+- Optional the size of the mask cutout (`EP_mask_x`, `EP_mask_y`) {float}
+- Paste can be split into regular grid with (`EP_num_paste_pads`) {[int (x), int (y)]}
+  - Paste coverage multiplicator (`EP_paste_coverage`) {float (0..1), default=0.65}
+
+### Thermal vias
+A package with exposed pad can generate a version with thermal vias. This will be generated in addition to the normal version.
+``` yaml
+  thermal_vias:
+    # thermal via version parameters
+```
+- Number of vias generated in the regular grid (`count`) {[int (x), int (y)]}
+- Final hole size (`drill`) {float}
+- Optional grid (`grid`) {[float (x), float (y)]}
+  - Auto generated if not given (outermost pad will touch pad edge.)
+- Optional paste coverage overwrite (`EP_paste_coverage`) {float (0..1)}
+  - Increase compared to non thermal via version to combat solder loss through wicking.
+- Should paste avoid vias? (`paste_avoid_via`) {bool}
+  - Can lead to math exceptions (reduce paste coverage)
+  - Clearance between via hole and paste controlled with (`paste_via_clearance`) {float}
+- Number paste pads
+  - given as count value (`EP_num_paste_pads`) {[int (x), int (y)]}
+  - More specific as between vias and outside ring (`paste_between_vias`, `paste_rings_outside`)
 
 
 ## Dimension parameter format
 
 ### String based
 
-The parameter can be given as a string of the format. Whitespace is ignored.
+The parameter can be given as a string of one of the following formats (whitespace characters are ignored).
```

Comment:
I think 'in one of' instead of 'of one of'.

Reply:
fixed